plugin/pkg/scheduler/core/generic_scheduler.go

```diff
@@ -474,6 +477,10 @@ func podFitsOnNode(
 				if !fit {
 					// eCache is available and valid, and predicates result is unfit, record the fail reasons
 					failedPredicates = append(failedPredicates, reasons...)
+					// if alwaysCheckAllPredicates is false, short circuit all predicates when one predicate fails.
```

Comment:
TODO here https://github.com/kubernetes/kubernetes/pull/56926/files#diff-a9420b7e5a53a445d54a716d341874abR458 can be removed now I think.

Reply:
That's ALL RIGHT.

Reply:
Ready for your review.